src/boot/efi/stdclib.c

```diff
@@ -0,0 +1,74 @@
+/* Public domain */
+/* Taken from https://clc-wiki.net/wiki */
+#include <stdint.h>
```

Comment:
so i really hope that this doesn't create conflicts with other code wanting to define these symbols.

Please rename this C file though, i.e. stdclib-glue.c or so, emphasizing this is just glue between std C and EFI libs. And make everything (at least where this is possible) really just wrappers around the gnuefi calls, i.e. stuff from https://github.com/vathpela/gnu-efi/blob/master/lib/str.c and related files.